<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="0" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-10-19T18:08:46 IST" name="Default suite" finished-at="2024-10-19T18:08:47 IST" duration-ms="932">
    <groups>
    </groups>
    <test started-at="2024-10-19T18:08:46 IST" name="Default test" finished-at="2024-10-19T18:08:47 IST" duration-ms="932">
      <class name="com.testng.OpenMRSRunner">
        <test-method is-config="true" signature="extendReportStartup()[pri:0, instance:com.testng.OpenMRSRunner@7b98f307]" started-at="2024-10-19T18:08:43 IST" name="extendReportStartup" finished-at="2024-10-19T18:08:46 IST" duration-ms="2753" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- extendReportStartup -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.testng.OpenMRSRunner@7b98f307]" started-at="2024-10-19T18:08:46 IST" name="setUp" finished-at="2024-10-19T18:08:46 IST" duration-ms="543" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[ERROR : FILNOTFOUND EXCEPTION OCCURE DURING PROPERTY FILE LOADING]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: ERROR : FILNOTFOUND EXCEPTION OCCURE DURING PROPERTY FILE LOADING
at org.junit.Assert.fail(Assert.java:89)
at com.property.FileReaderManagerMRS.setupProperty(FileReaderManagerMRS.java:23)
at com.property.FileReaderManagerMRS.getDataProperty(FileReaderManagerMRS.java:30)
at com.testng.OpenMRSRunner.setUp(OpenMRSRunner.java:29)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:527)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="validLogin()[pri:0, instance:com.testng.OpenMRSRunner@7b98f307]" started-at="2024-10-19T18:08:47 IST" name="validLogin" finished-at="2024-10-19T18:08:47 IST" duration-ms="1" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[ERROR : FILNOTFOUND EXCEPTION OCCURE DURING PROPERTY FILE LOADING]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: ERROR : FILNOTFOUND EXCEPTION OCCURE DURING PROPERTY FILE LOADING
at org.junit.Assert.fail(Assert.java:89)
at com.property.FileReaderManagerMRS.setupProperty(FileReaderManagerMRS.java:23)
at com.property.FileReaderManagerMRS.getDataProperty(FileReaderManagerMRS.java:30)
at com.testng.OpenMRSRunner.setUp(OpenMRSRunner.java:29)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:527)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validLogin -->
        <test-method is-config="true" signature="terminate()[pri:0, instance:com.testng.OpenMRSRunner@7b98f307]" started-at="2024-10-19T18:08:47 IST" name="terminate" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1729341527232" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- terminate -->
        <test-method is-config="true" signature="extentReportEnd()[pri:0, instance:com.testng.OpenMRSRunner@7b98f307]" started-at="2024-10-19T18:08:47 IST" name="extentReportEnd" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1729341527591" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- extentReportEnd -->
      </class> <!-- com.testng.OpenMRSRunner -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
